woof-code/3builddistro

```diff
@@ -508,9 +508,11 @@ if [ -n "$DOCS_URL" ]; then
 	if [ "$CACHE_PERSONAL_DOCS" = 'y' ]; then
```

Comment:
IMO this cache should be enabled by default, not the other way around. I know @peabee does local builds and prefers to cache as much as possible, and repeated downloads will kill the GitHub Actions traffic limits.